pkg/gitreceive/k8s_util.go

```diff
@@ -25,6 +24,11 @@ const (
 	dockerSocketPath = "/var/run/docker.sock"
 )
 
+var (
+	slugBuilderImage   = getenv("SLUGBUILDER_IMAGE_NAME", "quay.io/deisci/slugbuilder:v2-beta")
+	dockerBuilderImage = getenv("DOCKERBUILDER_IMAGE_NAME", "quay.io/deisci/dockerbuilder:v2-beta")
```

Comment:
also, the change I mentioned above will require that the values get passed down into the `dockerBuilderPod` and `slugBuilderPod` `func`s, so can you add to appropriate tests to ensure that the correct images are set?